.github/labeler.yml

```diff
@@ -11,8 +11,8 @@ io:
   - dask/dataframe/io/**/*
 
 documentation:
-  - dask/docs/*
-  - dask/docs/**/*
+  - docs/*
+  - docs/**/*
```

Comment:
Whoops -- thanks for fixing

Reply:
I just had a duh! moment. Thanks for fixing this. : )